arbnode/transaction_streamer.go

```diff
@@ -1481,6 +1481,14 @@ func (s *TransactionStreamer) setEspressoPendingTxnsPos(batch ethdb.KeyValueWrit
 }
 
 func (s *TransactionStreamer) SubmitEspressoTransactionPos(pos arbutil.MessageIndex, batch ethdb.Batch) error {
+	submitted, err := s.getEspressoSubmittedPos()
+	if err != nil && !dbutil.IsErrNotFound(err) {
+		return err
+	}
+
+	if pos == submitted {
+		return fmt.Errorf("Submitting the same position messages")
```

Comment:
I think something like "message at position %n already submitted" is a clearer message. Also will this log an error? If this is expected and we can handle it we shouldn't log an error because that might alarm anyone deploying the code that something is wrong.

Reply:
I think this is not a real error. However this function can only return an error or nil.
Compared to returning nil, returning error is more suitable. Your opinion is absolutly right, and I am thinking we can consider clearing the logs after all urgent things are basically done.